tests/installation/skip_registration.pm

```diff
@@ -21,7 +21,10 @@ use registration;
 use utils qw/ensure_fullscreen/;
 
 sub run() {
-    assert_screen([qw/scc-registration yast2-windowborder-corner/], 100);
+    if (!check_screen([qw/scc-registration yast2-windowborder-corner/], 210)) {
+        record_soft_failure 'bsc#990254';
+        assert_screen([qw/scc-registration yast2-windowborder-corner/], 100);
+    }
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).